style.css

```diff
@@ -16,38 +16,52 @@
 
 .name {
   font-family: 'Roboto', sans-serif;
-  font-size: 18px;
+  font-size: 20px;
   font-weight: 700;
   position: relative;
   top: 50px;
   left: 10px;
+  margin-left: 15px;
+  margin-top: -30px;
 }
 
 .menu {
   position: absolute;
-  top: 50px;
+  top: 55px;
   left: 330px;
+  margin-top: -30px;
+}
+
+.men {
+  display: none;
 }
 
 .section1 {
-  background-color: black;
+  background-color: #1c1a19;
   width: 375px;
-  margin-top: 120px;
+  margin-top: -30px;
   margin-left: -8px;
```

Comment:
- [ ] Kindly make sure the content of your design fills the browser viewports width so as to make your design look better and more professional 🙏 

![image](https://user-images.githubusercontent.com/74814780/206458570-65e7e63a-a2be-4424-95e0-d9e22bb0734a.png)